libbeat/logp/level.go

```diff
@@ -0,0 +1,74 @@
+package logp
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"go.uber.org/zap/zapcore"
+)
+
+// Level is a logging priority. Higher levels are more important.
+type Level int8
+
+// Logging levels.
+const (
+	DebugLevel Level = iota - 1
+	InfoLevel
+	WarnLevel
+	ErrorLevel
+	CriticalLevel // Critical exists only for config backward compatibility.
```

Comment:
Do we use `Critical` somewhere in our beats code?

Reply:
We previously supported `critical` as a config option so I had to carry it forward (`logging.level: critical`). I removed it from the docs.